content/ja/docs/contributing/_index.md

```diff
@@ -1,26 +1,45 @@
 ---
 title: コントリビューション
-description: OpenTelemetryドキュメントへのコントリビューション方法について学ぶ。
 aliases: [/docs/contribution-guidelines]
 weight: 980
-default_lang_commit: a4f4d8b07313532d44d4041183c515875ad4d60d
+default_lang_commit: 24146bd1368e4c6082c7d6077efd29dba0d51055
 ---
 
+OpenTelemetry のドキュメントとWebサイトのコントリビューションに興味を持っていただきありがとうございます。
+
+## 今すぐ飛び込んでみよう! {#jump-right-in}
+
+何がしたいですか?
+
+- タイプミスやその他の簡単な修正を行う場合は、[GitHub を利用したコンテンツの提出](pull-requests/#changes-using-github)を確認してください
+- より重要なコントリビューションを行う場合は、このセクション内の以下のページからお読みください。
+  - [Prerequisites]
+  - [Issues]
+  - [Submitting content]
```

Comment:
If you wish to translate the link text you can do so like this:

```suggestion
  - [前提条件][Prerequisites]
  - [問題][Issues]
  - [コンテンツの提出][Submitting content]
```

Reply:
@open-telemetry/docs-ja-approvers ☝️

Reply:
@chalin 

I agree with your idea because I've been already asked to do a similar advice.

https://github.com/open-telemetry/opentelemetry.io/pull/6179/commits/7682dca8e3dbd8c415dcc7680be7e7007d4668ca